src/Command/ModelCommand.php

```diff
@@ -404,6 +406,112 @@ public function findTableReferencedBy(TableSchemaInterface $schema, string $keyF
         return null;
     }
 
+    /**
+     * Checks whether the given source and target table names are sides
+     * of a possible many-to-many relation.
+     *
+     * @param string $sourceTable The source table name.
+     * @param string $targetTable The target table name.
+     * @return bool
+     */
+    public function isPossibleBelongsToManyRelation(string $sourceTable, string $targetTable): bool
+    {
+        $tables = $this->listAll();
+
+        $pregTableName = preg_quote($sourceTable, '/');
+        $pregPattern = "/^{$pregTableName}_|_{$pregTableName}$/";
+
+        if (preg_match($pregPattern, $targetTable) === 1) {
+            $possibleBTMTargetTable = preg_replace($pregPattern, '', $targetTable);
+            if (in_array($possibleBTMTargetTable, $tables)) {
```

Comment:
can we use true as 3rd arg here always?

Reply:
Probably, it shouldn't make a difference with the return value of `preg_replace()` here, it will be either a string or `null`.